apps/webClient/src/components/BulbIcon/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+interface Props {
+  height?: number;
+  width?: number;
+}
+
+const BulbIcon: React.FC<Props> = ({ height = 70, width }) => {
+  const svgProps = {};
+  if (height) {
+    Object.assign(svgProps, { height });
+  }
+  if (width) {
+    Object.assign(svgProps, { width });
+  }
+  return (
+    <svg
+      id="prefix__Layer_1"
+      x={0}
+      y={0}
+      viewBox="0 0 59.3 58.14"
+      xmlSpace="preserve"
+      {...svgProps}
+    >
+      <path
+        style={{ fill: "#23221c" }}
```

Comment:
instead of using `style` here, just do what's done on line 32:

`<path fill="#hex"...`